components/combobox.tsx

```diff
@@ -1,214 +1,109 @@
-import { useState, useRef, useEffect, ChangeEvent, KeyboardEvent } from "react";
-import {
-  useFloating,
-  offset,
-  flip,
-  shift,
-  autoUpdate,
-  Placement,
-  useDismiss,
-  useInteractions,
-} from "@floating-ui/react";
-import {
-  RiArrowDownSLine,
-  RiArrowUpSLine,
-  RiCloseLine,
-} from "@remixicon/react";
+import { CSSProperties, HTMLAttributes, MutableRefObject, Ref } from "react";
+import { Selectbox } from "./selectbox";
 
 interface ComboboxProps {
   options: string[];
+  type?: "default" | "calendarItems";
```

Comment:
when is this `"calendarItems"` used? in general, the combobox shouldn't even care/know if it is calendar items or etc...